bin/commands.sh

```diff
@@ -34,8 +34,8 @@ if [ "$1" = "ghactionsbootstrap" ]; then
   echo "Entering .docs"
   cd .docs
 
-  echo "Running npm install inside .docs"
-  npm install
+  echo "Running yarn install inside .docs"
+  yarn install
```

Comment:
Since this is the `ghactionsbootstrap` command, I think we should run `yarn install --pure-lockfile` here

Reply:
Thanks for the feedback. The yarn install command has been updated to reflect your feedback.